iis-schemas/src/main/avro/eu/dnetlib/iis/export/DocumentToSoftwareUrls.avdl

```diff
@@ -0,0 +1,18 @@
+@namespace("eu.dnetlib.iis.export.schemas")
+protocol IIS{
+
+    record SoftwareUrl {
+        // software URL	
+        string softwareUrl;
+        // repository name
+        string repositoryName;
+        // find more details on `confidenceLevel` constraints in eu/dnetlib/iis/README.markdown file.
```

Comment:
Referenced file was placed in `iis-schemas/src/main/avro/eu/dnetlib/iis/README.markdown` and it was available only in git sources.

I've moved it into `iis-schemas/src/main/resources/eu/dnetlib/iis/README.markdown` so it become part of jar artifact.